webapi/webapi-webgl-khronos-tests/webgl/khronos/COPYING

```diff
@@ -93,6 +93,63 @@ function testPassed(assertName, name) {
  }
 
 
+Remove unused WebGL test cases:
```

Comment:
Just curious why these tests are unused? Are they used in Khronos conformance test suite? If not, why Khronos keeps them?